rbac/management/tenant_service/v2.py

```diff
@@ -262,6 +269,27 @@ def _bootstrap_tenant(self, tenant: Tenant) -> BootstrappedTenant:
 
         return BootstrappedTenant(tenant, mapping, default_workspace=default_workspace, root_workspace=root_workspace)
 
+    def _replicate_bootstrap(self, tenant: Tenant, mapping: TenantMapping):
+        """Replicate the bootstrapping of a tenant."""
+        built_in_workspaces = Workspace.objects.filter(
+            tenant=tenant, type__in=[Workspace.Types.ROOT, Workspace.Types.DEFAULT]
+        )
+        root = next(ws for ws in built_in_workspaces if ws.type == Workspace.Types.ROOT)
```

Comment:
With the above, we might also be able to have a manager add to the query to make root and default return ordering deterministic so we could just have assign them in one line, but again, not necessary here.